src/qc/calculate_qc_metrics/script.py

```diff
@@ -34,28 +42,42 @@ def setup_logger():
 # END TEMPORARY WORKAROUND setup_logger
 logger = setup_logger()
 
+def count_nonzero(layer, axis):
```

Comment:
could you add a comment or two that this performs the same computations as running `layer.getnnz(...)` but solves an issue related to xxxx and yyyy?

Reply:
Done!